tests/main/document-portal-activation/fake-document-portal.py

```diff
@@ -43,14 +46,15 @@ def main(argv):
         path="/org/freedesktop/DBus/Local",
         dbus_interface="org.freedesktop.DBus.Local")
 
-    portal = DocPortal(bus, OBJECT_PATH, logfile, errorfile)
+    DocPortal(bus, OBJECT_PATH, logfile, errorfile)
```

Comment:
Hm, actually - you had to rever the other `portal = PortalImpl(bus, OBJECT_PATH, None)` - do we need this here as well?

Reply:
That's weird, do we have the log from before the branch got force pushed?

Reply:
@mvo5 I don't know since the test passed, but again not being an Python expert it's not clear to me why this was necessary in the first place. 

@bboozzoo I re-ran the old changes through spread on qemu and here's the log: https://pastebin.ubuntu.com/p/28H42WmvyF/

Reply:
This looks bad, you are creating an object which is instantly collected. I doubt this is what you want.